Description: Patch to fix libs installation, plus amend CFLAGS and LDFLAGS
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
-prefix = /usr/local
+prefix = $(DESTDIR)/usr
 exec_prefix = $(prefix)
-libdir = $(exec_prefix)/lib
+libdir = $(exec_prefix)/lib/$(DEB_HOST_MULTIARCH)
 bindir = $(exec_prefix)/bin
 includedir = $(prefix)/include
 pkgcfgdir = $(libdir)/pkgconfig
@@ -14,9 +14,9 @@ LEXCLEX = flex -8 --prefix=lexc
 LEXIFACE = flex -8 --prefix=interface
 LEXCMATRIX = flex -8 --prefix=cmatrix
 RM = /bin/rm -f
-LDFLAGS = -lreadline -lz -ltermcap
-FLOOKUPLDFLAGS = libfoma.a -lz
-CFLAGS = -O3 -Wall -D_GNU_SOURCE -std=c99 -fvisibility=hidden -fPIC
+LDFLAGS = $(shell dpkg-buildflags --get LDFLAGS) -lreadline -lz -ltermcap -Wl,-z,relro
+FLOOKUPLDFLAGS = $(shell dpkg-buildflags --get LDFLAGS) libfoma.a -lz
+CFLAGS = $(shell dpkg-buildflags --get CPPFLAGS) $(shell dpkg-buildflags --get CFLAGS) -Wall -D_GNU_SOURCE -std=c99 -fcommon -fvisibility=hidden -fPIC -Wl,-z,relro
 FOMAOBJS = foma.o stack.o iface.o lex.interface.o
 LIBOBJS = int_stack.o define.o determinize.o apply.o rewrite.o lexcread.o topsort.o flags.o minimize.o reverse.o extract.o sigma.o io.o structures.o constructions.o coaccessible.o utf8.o spelling.o dynarray.o mem.o stringhash.o trie.o lex.lexc.o lex.yy.o lex.cmatrix.o regex.o
 
